cartographer_grpc/framework/rpc_handler.h

```diff
@@ -27,7 +27,7 @@ namespace framework {
 class Rpc;
 class RpcHandlerInterface {
  public:
-  void SetRpc(Rpc* rpc);
+  virtual void SetRpc(Rpc* rpc) = 0;
```

Comment:
If a class has a virtual function, it should have a virtual destructor.

Reply:
Done.